### DISCUSSION TITLE
Install assistance

### DISCUSSION THREAD

**Discussion by stevecopeza** [johanmeijer/grott#64] [General]
Hi @johanmeijer , as much as I’ve read what I can around grott I’m not confident to install this on my HA without breaking something. Would you have capacity to assist, I 7nderstand there’s likely to be a charge

**Comment by johanmeijer** [comment]
Hi. I am not a home assistant expert and have not  installed Grott together with HA.

Maybe you van have a look here: https://github.com/wjansenw/home-assistant-add-on. 

It should not be that difficult.

**Reply by JazekerXX** [reply]
this link is not available anymore..

**Reply by PeterKawa** [reply]
Just look for 'grott' in HACS store
